PixelHunter1995/SceneLib/SceneParser.cs

```diff
@@ -95,9 +95,6 @@ public static Scene ParseSceneXml(string sceneXmlPath)
                     if (player == null)
                     {
                         player = new Player(50, 50);
```

Comment:
now when not added to all list maybe player should be created somewhere else since we are not parsing it from scene xml

Reply:
I think we should parse it from the scene in the future though. Its position at least.